docs/developer/tutorials/snmp/tools.md

```diff
@@ -0,0 +1,42 @@
+# Tools
+
+## Using `tcpdump` with SNMP
+
+`tcpdump` shows the exact request and response content of SNMP `GET`, `GETNEXT` and other SNMP calls.
```

Comment:
```suggestion
The `tcpdump` command shows the exact request and response content of SNMP `GET`, `GETNEXT` and other SNMP calls.
```
(in part so we're not starting a sentence with a lowercase letter)

Reply:
Thanks @kayayarai